server/src/main/java/fr/uga/l3miage/tp1/exo2/models/NetflixUserEntity.java

```diff
@@ -0,0 +1,29 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import fr.uga.l3miage.tp1.exo2.enums.Sex;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "netflix_user")
+public class NetflixUserEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    Integer id;
+
+    String uuid;
+
+    String lastname;
+
+    String firstname;
+
+    @Enumerated(EnumType.STRING)
+    Sex sex;
```

Comment:
il manque une contrainte sur la taille
```suggestion
    @Enumerated(EnumType.STRING)
    @Column(name="sex", length=2)
    Sex sex;
```